### DISCUSSION TITLE
Grüner Punkt abschaltbar?

### DISCUSSION THREAD

**Discussion by warped-rudi** [tbnobody/OpenDTU#1659] [General]
Seit v24.1.14 erscheint in der Liveansicht eine grün blinkende Animation, deren Sinn sich mir nicht erschließt. Kann man das abschalten? Es lenkt unnötig von den anderen dargebotenen Informationen ab und bringt Unruhe in die Anzeige. Sprich: Es nervt.

**Comment by Marc9485** [comment]
Grün / verbunden
Rot / nicht verbunden

**Reply by stefan123t** [reply]
Der `onlineMarker` wird je nach Zustand des Flags `isWebsocketConnected` rot oder grün dargestellt.
Es gibt also an ob der Browser eine WebSocket Verbindung mit dem Web Server auf der OpenDTU bestehen hat.

Vielleicht könnte man die Icon-/Bild- bzw. Animations-Semantik einfach umkehren ?

* Grün : einfach nur da, evtl. noch ganz langsam - und nur ganz leicht - größer und kleiner pulsieren.
![image](https://github.com/tbnobody/OpenDTU/assets/11159753/3a673176-1db1-46ca-9732-0a10a4009519)

* Rot : das bisherige Verhalten bei dem die Bubble platzt oder noch besser in sich zusammenfällt.
![image](https://github.com/tbnobody/OpenDTU/assets/11159753/9e1e86d1-be93-4759-b92c-73b38a5911e4)

Damit sollte man auch den eher ruhigeren Naturen unter uns Nutzern entgegenkommen,
ohne epileptische Anfälle vor dem Bildschirm auszulösen ;)

**Reply by stefan123t** [reply]
BTW @tbnobody ich habe gerade mal testweise die OpenDTU abgeklemmt (ich wollte den zweiten, roten Screenshot von der Bubble machen) und das hat das Flag `isWebsocketConnected` gar nicht gekümmert. Evtl ist da noch etwas nicht ganz so wie es sein soll ?

**Comment by warped-rudi** [comment]
Danke für die Info! Aber gäbe es dafür nicht eine dezentere Möglichkeit der Anzeige? Z.B. Graufärben der Panelüberschriften, Information im Statusbanner, Schriftfarbe generell von schwarz ->hellgrau? Auf einem Mobilgerät fällt dieses Blinken ja nicht so auf, da man in der Regel nur ein Panel zu Gesicht bekommt bzw. vom entsprechenden Bildbereich wegscrollt. In einem Desktopbrowser in Vollansicht dagegen ist man ständig mental damit beschäftigt, diese Blinken in der linken oberen Ecke auszublenden.

**Comment by MichlW** [comment]
Verbunden womit? Das erschließt sich doch nicht.
Die Verbindung zu den Wechselrichtern sieht man doch bereits in den "Tabs".

Sinnvoller aus Usabilitysicht wäre es nur dann einen Indikator anzuzeigen, wenn eine Verbindung nicht besteht.

**Reply by warped-rudi** [reply]
Es geht vermutlich nicht um die Verbindung der DTU zum Wechselrichter, sondern um die zwischen Browser und OpenDTU. Diese wird zur Aktualisierung der Live-Daten benötigt. Bricht sie ab, sieht man statische, veraltete Werte. In dieser Situation ist ein Neuladen der Seite nötig.

**Reply by MichlW** [reply]
Danke dir. Mir schwante schon, dass es was damit zu tun haben muss, ich wollte es aber bewusst etwas provokativ darstellen.
Ich bewerbe dennoch erneut meinen Vorschlag, nur ein (Fehler-)Symbol im Problemfall anzuzeigen - und nicht generell. Die Erwartungshaltung sollte nämlich auch ohne grünen Punkt sein, dass "Live Daten" auch live sind.
